string command
float64[] Qtt_hat
float64[] Vt_hat
string group_name
int64 t
---
float64[] xt_hat
float64[] Qt_hat
float64[] xt
float64[] Qt
float64[] x_next
float64[] Q_next
float64 if_collide